### PR TITLE
Fix Trivy issues in Dockerfile.sources

### PR DESCRIPTION
**Pull request by Mionsz**
Fix Trivy issues in Dockerfile.sources
- add mock healthcheck
- add `tiber` user
- fix missing `--no-install-recommends`

Update build_tiber.yml
- fix cache location
- remove not safe runner data removal

Update Dockerfile, load env file from repository.
- removed arguments passing using custom script.

Update build.sh and Dockerfile
- default env is loaded from versions.env
- when using build.sh any file can be specifiedm .temp.env will be populated
- allows build without the scripting in the process

Ci/Cd: Added Docker Buildx Caching:
- each stage have now a cache reference added
- each image can now be pushed to namespaced registry
- removed unused architectures and QEMU builder

Update tags for image push:
- fixed docker cache-from
- fixed docker cache-to
- fixed login issues

Remove no longer used VSR Patches
Both are included in wheel repository.

Fix JpegXS build process and nproc number
- Added valid install directory to omit LD_LIBRARY_PATH
- Added dawid.wesierski@intel.com rto authors
- Fix dockerfiles version label to reflect repository version.

FIX: Patches cleanup and script optimizations
- number applied as prefix for sorting by name
- use patch native method instead of git
- other optimizations

ADDED: More simplifications in build system.
- unified pkg-config path
- unified install prefix path

FIX: Sort Packages Lexicographically, Ascending Order.
- makes track of packages easy

FIX: Minor check script adjustment

### CODE REVIEW

**Review by zLukas — DISMISSED** [review]

**Review by soopel — APPROVED** [review]
Lgtm

File: .github/workflows/publish_tiber.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  push:
+    branches:
+    - 'main'
+    - 'release-*'
+    tags:
+    - 'v*'
+
+env:
+  BUILD_TYPE: "Release"
+  DOCKER_IMAGE_NAME: "ghcr.io/openvisualcloud/intel-tiber-broadcast-suite/tiber-broadcast-suite"
```
Comment (zLukas) [inline .github/workflows/publish_tiber.yaml:16]: are we allowed to use ghrc.io ?
it's internal github registry

**Review by zLukas — CHANGES_REQUESTED** [review]

File: .github/workflows/build_tiber.yml [hunk 2]
```diff
@@ -9,21 +9,22 @@ on:
 
 permissions:
   contents: read
-  security-events: write
 
 env:
   BUILD_TYPE: "Release"
-  DOCKER_IMAGE_NAME: "docker.io/ovc/tiber-broadcast-suite"
+  DOCKER_IMAGE_NAME: "ghcr.io/openvisualcloud/intel-tiber-broadcast-suite/tiber-broadcast-suite"
```
Comment (zLukas) [inline .github/workflows/build_tiber.yml:15]: do we discussed where this image will be pushed at all ?

**Review by zLukas — CHANGES_REQUESTED** [review]

File: .github/workflows/build_tiber.yml (hunk 2 above)
Comment (Mionsz) [reply .github/workflows/build_tiber.yml:15]: This the only possibility - `ghcr.io` / `namespace` / `repository` / `artifact-name` : `tag`.
This ends up as a summary and as it is a free tier not the paid one, we does not see it anywhere other then as our artifacts: 
image

https://github.com/OpenVisualCloud/Media-Communications-Mesh/actions/runs/11962570821

File: .github/workflows/publish_tiber.yaml (hunk 1 above)
Comment (Mionsz) [reply .github/workflows/publish_tiber.yaml:16]: This approach is the only solution as viewed from Docker and docker-actions know-how point of view. I am not yet fully utilizing it as one should but this must be done step by step.

Comment (zLukas) [reply .github/workflows/publish_tiber.yaml:16]: at this moment, I suggest to disable this action, code can stay here, but do not execute it yet

Comment (Mionsz) [reply .github/workflows/publish_tiber.yaml:16]: The push action is disbaled. Caching stays at current minimal level - as same as in the MCM

**Review by soopel — APPROVED** [review]
Approved

**Review by zLukas — APPROVED** [review]

**Pull request merged**